Messages/MessageElement.php

```diff
@@ -77,13 +79,23 @@ public function __construct($title, $subtitle, $image_url = '', $buttons = [], $
      */
     public function getData()
     {
-        $result = [
-            'title' => $this->title,
-            'subtitle' => $this->subtitle,
-            'item_url' => $this->url,
-            'image_url' => $this->image_url,
-            'default_action' => $this->default_action
-        ];
+        
+         if (!empty($default_action)) {
+            $result = [
+                'title' => $this->title,
+                'subtitle' => $this->subtitle,
+                'item_url' => $this->url,
+                'image_url' => $this->image_url,
+                'default_action' => $this->default_action
+            ];
+        } else {
+            $result = [
+                'title' => $this->title,
+                'subtitle' => $this->subtitle,
+                'item_url' => $this->url,
+                'image_url' => $this->image_url,
+            ];
+        }
```

Comment:
```php
$result = [
    'title' => $this->title,
    'subtitle' => $this->subtitle,
    'item_url' => $this->url,
    'image_url' => $this->image_url,
];

if (!empty($default_action)) {
    $result['default_action'] = $this->default_action;
}
```

Reply:
Wolzogens code is cleaner. Any pull request for this?